cibuildwheel/bashlex_eval.py

```diff
@@ -45,14 +45,14 @@ def evaluate(
 def evaluate_node(node: bashlex.ast.node, context: NodeExecutionContext) -> str:
     if node.kind == "word":
         return evaluate_word_node(node, context=context)
-    elif node.kind == "commandsubstitution":
+    if node.kind == "commandsubstitution":
         node_result = evaluate_command_node(node.command, context=context)
         # bash removes training newlines in command substitution
         return node_result.rstrip()
-    elif node.kind == "parameter":
+    if node.kind == "parameter":
         return evaluate_parameter_node(node, context=context)
-    else:
-        raise ValueError(f'Unsupported bash construct: "{node.kind}"')
+
+    raise ValueError(f'Unsupported bash construct: "{node.kind}"')
```

Comment:
Yeah, 90% of these if..elif refactors make it harder to read IMO.

Reply:
The final expression is "special", it's the "nothing matched" case, so this highlights it as such. It also actually makes the code easier to lint; see https://github.com/tomerfiliba/plumbum/pull/584/files#diff-f8094a3c5928e29d7188c84488925d2728abc59163ba387451cca5a8cdcd54e8 - besides removing a lot of white space, "guard" style fixed an issue parsing the code that make liners think there was a way to return without a return statement.

Some of them are neither better nor worse, IMO, but this is one that's a tiny bit better. If we could use match statements, that would be even better, but we can't, yet.

Reply:
> Some of them are neither better nor worse, IMO, but this is one that's a tiny bit better. If we could use match statements, that would be even better, but we can't, yet.

agreed